package.json

```diff
@@ -235,14 +241,8 @@
   },
   "husky": {
     "hooks": {
-      "pre-commit": "npm run format:fix && npm run license:fix && lint-staged",
+      "pre-commit": "npm run format:fix && npm run license:fix",
       "commit-msg": "commitlint -E HUSKY_GIT_PARAMS"
     }
-  },
-  "lint-staged": {
-    "src/clr-core/**/*.{js,ts}": [
-      "npm run core:build:api",
-      "git add"
-    ]
```

Comment:
Lint staged is no longer needed since we no longer need to generate markdown files for core API docs